src/commands/experiments-list.ts

```diff
@@ -0,0 +1,25 @@
+import { Command } from "../command";
+import Table = require("cli-table");
+import * as experiments from "../experiments";
+import { partition } from "../functional";
+import { logger } from "../logger";
+
+export const command = new Command("experiments:list").action(() => {
+  const table = new Table({
+    head: ["Enabled", "Name", "Description"],
+    style: { head: ["yellow"] },
+  });
+  const [enabled, disabled] = partition(Object.entries(experiments.ALL_EXPERIMENTS), ([name]) => {
+    return experiments.isEnabled(name as experiments.ExperimentName);
+  });
+  for (const [name, exp] of enabled) {
+    table.push(["y", name, exp.shortDescription]);
+  }
+  for (const [name, exp] of disabled) {
+    if (!exp.public) {
+      continue;
+    }
+    table.push(["n", name, exp.shortDescription]);
+  }
+  logger.info(table.toString());
```

Comment:
One detail that can be punted on until later: for `--json` (machine readable) output, we could export the table as a JSON-ish object (just return an object in this method). Would that be easy enough to do?

Reply:
I couldn't get this to work. You'll have to teach me how --json works with the commands library